core/domain/skill_services.py

```diff
@@ -747,6 +749,10 @@ def update_skill(committer_id, skill_id, change_list, commit_message):
     skill = apply_change_list(skill_id, change_list, committer_id)
     _save_skill(committer_id, skill, commit_message, change_list)
     create_skill_summary(skill.id)
+    taskqueue_services.defer(
```

Comment:
Shouldn't we do this only if misconception is edited?

Reply:
Oh yes! Added a check. Done.